portal_api.lua

```diff
@@ -2166,22 +2166,20 @@ function nether.unregister_portal(name)
 end
 
 function nether.register_portal_ignition_item(item_name, ignition_failure_sound)
-
+	old_on_place = minetest.registered_items[item_name].on_place or minetest.item_place
```

Comment:
```suggestion
	local old_on_place = minetest.registered_items[item_name].on_place or minetest.item_place
```

You'd pollute the global `_G` table.